src/Avengers.js

```diff
@@ -0,0 +1,69 @@
+import './Avengers.css';
+import axios from 'axios';
+import { useEffect, useState } from "react";
+import AvengersDisplayPhotos from './AvengersDisplayPhotos';
+import GoBack from './GoBack';
+
+const Avengers = () => {
+
+    //call useState hook to hold the array back from my api - destructure variable and function 
+    const [charData, setCharData] = useState(null); //expecting an array of objects, so I initialize with null because it doesn't exist until user selection
+     // Character ID -> coming from select
+    const [charId, setCharId] = useState('');
+
+  
+    useEffect(() => {
+
+      if(charId){
+    
+      axios({
+        baseURL: 'https://gateway.marvel.com/',
+        url: `v1/public/characters/${charId}`,
+        method: 'GET',
+        params: {
+          ts: 'thesoer',
+          apikey: 'f6a569770dbc42c8e49153f86a614843',
+          hash: '2b386c7f05318a6e66fed6f3416c4b2d',
+        }
+      }).then((apiData) => {
+    
+        setCharData(apiData.data.data.results)
+      })
+      }      
+    }, [charId])
+
+
+    //tracking change event on select el and form submission
+    const handleChange = (event) => {
+      console.log(event.target.value)
+      const value = event.target.value;
+      setCharId(value);
+    }
+
+    return (
+        <div className='avengersPage' >
+          <h1>Avengers</h1>
+          <GoBack />
```

Comment:
May want to add specific instructions for what the user would use the drop-down for. Maybe "Select your favourite character and get some more info!" or something like that.